packit_service/worker/testing_farm.py

```diff
@@ -207,15 +207,19 @@ def chroot2distro_arch(self, chroot: str) -> Tuple[str, str]:
         """Get distro and arch from chroot."""
         distro, arch = chroot.rsplit("-", 1)
 
-        epel_mapping = {
-            "epel-6": "centos-6",
-            "epel-7": "centos-7",
-        }
+        distros = self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
+        if distros:
+            # For now take just the first one
+            return distros[0], arch
```

Comment:
What about [assignment expression](https://www.python.org/dev/peps/pep-0572/)?

```suggestion
        if distros := self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
            # For now take just the first one
            return distros[0], arch
```

Reply:
Ah, actually black reformatted it into
```python
        if distros := self.job_config.metadata.targets_dict.get(chroot, {}).get(
            "distros"
        ):
```
in which case I think the old
```python
        distros = self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
        if distros:
```
is a bit nicer and easier to "parse". So I'd probably leave it as it is.